ci/includes/pinned-tools.sh

```diff
@@ -31,3 +38,13 @@ export LINTER="cleanupcode.exe"
 export MSBUILD="$(powershell  –ExecutionPolicy Bypass ./ci/find-msbuild.ps1)"
 
 export NUNIT3_CONSOLE="code_generator/packages/NUnit.ConsoleRunner.3.8.0/tools/nunit3-console.exe"
+
+function unpackTo() {
+  local SOURCE=$1
+  local TARGET=$2
+
+  mkdir -p "${TARGET}"
+  unzip -o -q "${SOURCE}" -d "${TARGET}"
+}
+
+PINNED_CORE_SDK_VERSION="13.0.1"
```

Comment:
I think it would be better to move this into its own file to avoid any kind of merge conflicts later